### PR TITLE
Update PARs upload UI

### PR DESCRIPTION
**Pull request by sgrowe**
Haven't finished the whole journey yet, but it's probably worth reviewing & merging what there is now and add the rest in a later PR.

Currently there is:
- Page to choose whether you want to add a new PAR or update an existing one
- Form for adding product details
- Can add multiple substances
- Can add multiple products

Still to do:
- Add file page
- Review form data page
- Actual form submission
- Form submission success/error page
- Remove added products (no UI designs for that as yet though?)
- Auth stuff (in #757)
- Update an existing document form (in #760)

### CODE REVIEW

File: .github/workflows/pars-upload-branch.yaml [hunk 1]
```diff
@@ -22,9 +22,31 @@ jobs:
         with:
           node-version: "13.11"
 
+      # Based on https://github.com/actions/cache/blob/master/examples.md#node---yarn
+      - name: Get yarn cache path
```
Comment (rjkerrison) [inline .github/workflows/pars-upload-branch.yaml:26]: awesome 👏

File: medicines/pars-upload/src/header.js [hunk 2]
```diff
@@ -20,7 +21,7 @@ export const Header = () => (
           <span className="govuk-header__logotype">
             <MhraLogo />
           </span>
-          MHRA PARs upload homepage
+          <ScreenReaderOnly>MHRA PARs upload homepage</ScreenReaderOnly>
```
Comment (rjkerrison) [inline medicines/pars-upload/src/header.js:24]: 👍

**Comment by sgrowe** [comment]
I haven't finished the whole form yet but this PR is getting fairly big already so it probably makes sense to review and merge this now and then we can raise another PR later.

File: medicines/pars-upload/cypress/integration/tests.js [hunk 3]
```diff
@@ -1,8 +1,68 @@
...
+    cy.findByText('Upload a new document').click();
+
+    cy.findByText('Continue').click();
+
+    cy.findByText('New Public Assessment Report').should('exist');
+  });
+});
+
+describe('PARs upload form', () => {
+  it('can add multiple substances', () => {
```
Comment (majikandy) [inline medicines/pars-upload/cypress/integration/tests.js:16]: 👍

**Pull request merged**